Cannot read property 'setList_4_g$' of undefined in WorkbenchListManager at clean startup

### System details

    RStudio Edition : Desktop or Server (dev)
    RStudio Version : 1.3.1058, master (1.4)
    OS Version      : macOS
    R Version       : 3.6.3

### Steps to reproduce the problem
- starting with a clean RStudio state (use `rstudio/scripts/clean-rstudio`)
- build and run `rserver-dev` / `ant devmode` OR `rstudio-dev` / `ant desktop`
- load IDE (in Chrome or desktop)
- open JavaScript console

### Describe the problem in detail
- in the JavaScript console, see:

```
WARNING: Exception occurred dispatching event: list_changed

Caused by: com.google.gwt.core.client.JavaScriptException: (TypeError) : Cannot read property 'setList_4_g$' of undefined
ConsoleLogger.java:33 TypeError: Cannot read property 'setList_4_g$' of undefined
    at WorkbenchListManager_1_g$.WorkbenchListManager.updateList [as updateList_2_g$] (WorkbenchListManager.java:106)
    at WorkbenchListManager$2_1_g$.WorkbenchListManager$2.onListChanged [as onListChanged_5_g$] (WorkbenchListManager.java:69)
    at ListChangedEvent_2_g$.ListChangedEvent.dispatch [as dispatch_1155_g$] (ListChangedEvent.java:58)
    at ListChangedEvent_2_g$.ListChangedEvent.dispatch [as dispatch_913_g$] (ListChangedEvent.java:56)
    at ListChangedEvent_2_g$.GwtEvent.dispatch [as dispatch_912_g$] (GwtEvent.java:76)
    at EventBus.dispatchEvent (EventBus.java:40)
    at HandlerManager$Bus_1_g$.SimpleEventBus.doFire [as doFire_1_g$] (SimpleEventBus.java:193)
    at HandlerManager$Bus_1_g$.SimpleEventBus.fireEvent [as fireEvent_43_g$] (SimpleEventBus.java:88)
    at EventBus_3_g$.HandlerManager.fireEvent [as fireEvent_42_g$] (HandlerManager.java:127)
    at EventBus_3_g$.EventBus.fireEvent [as fireEvent_45_g$] (EventBus.java:92)
    at EventBus_3_g$.EventBus.dispatchEvent [as dispatchEvent_14_g$] (EventBus.java:59)
    at ClientEventDispatcher_1_g$.ClientEventDispatcher.dispatchEvent [as dispatchEvent_16_g$] (ClientEventDispatcher.java:546)
    at ClientEventDispatcher$1_1_g$.ClientEventDispatcher$1.execute [as execute_1578_g$] (ClientEventDispatcher.java:235)
    at SchedulerImpl$Task.$executeRepeating (SchedulerImpl.java:46)
    at SchedulerImpl.runScheduledTasks (SchedulerImpl.java:179)
    at SchedulerImpl_1_g$.SchedulerImpl.flushPostEventPumpCommands [as flushPostEventPumpCommands_1_g$] (SchedulerImpl.java:347)
    at SchedulerImpl$Flusher_1_g$.SchedulerImpl$Flusher.execute [as execute_1578_g$] (SchedulerImpl.java:78)
    at SchedulerImpl.execute (SchedulerImpl.java:141)
    at Impl.apply (Impl.java:309)
    at Impl.entry0 (Impl.java:361)
    at Impl.java:78
    at callback_0_g$ (SchedulerImpl.java:196)
```

### Describe the behavior you expected
- no exceptions in normal use
- does not happen on subsequent runs (only during a clean run)

### Underlying cause

The rsession code defines `addins_mru`:

https://github.com/rstudio/rstudio/blob/e12991f7f8f37d70dfd0a382fd8c5276673101db/src/cpp/session/modules/SessionLists.cpp#L48

But the addins mru feature was removed here, possibly missing this bit: https://github.com/rstudio/rstudio/commit/20f8d04b68e99a1bd0dd1330e5f10397884496f5

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

--- a/src/cpp/session/modules/SessionLists.cpp
+++ b/src/cpp/session/modules/SessionLists.cpp
@@ -45,7 +45,6 @@ const char * const kProjectMru = kProjectMruList;
 const char * const kHelpHistory = "help_history_links";
 const char * const kUserDictioanry = "user_dictionary";
 const char * const kPlotPublishMru = "plot_publish_mru";
-const char * const kAddinsMru = "addins_mru";
 
 // path to lists dir
 FilePath s_listsPath;
@@ -298,7 +297,6 @@ Error initialize()
    s_lists[kHelpHistory] = 15;
    s_lists[kPlotPublishMru] = 15;
    s_lists[kUserDictioanry] = 10000;
-   s_lists[kAddinsMru] = 15;
 
    // monitor the lists directory
    s_listsPath = module_context::registerMonitoredUserScratchDir(

--- a/src/gwt/src/org/rstudio/studio/client/workbench/WorkbenchListManager.java
+++ b/src/gwt/src/org/rstudio/studio/client/workbench/WorkbenchListManager.java
@@ -17,6 +17,7 @@ package org.rstudio.studio.client.workbench;
 import java.util.ArrayList;
 import java.util.HashMap;
 
+import org.rstudio.core.client.Debug;
 import org.rstudio.studio.client.application.events.EventBus;
 import org.rstudio.studio.client.server.VoidServerRequestCallback;
 import org.rstudio.studio.client.workbench.events.ListChangedEvent;
@@ -103,7 +104,13 @@ public class WorkbenchListManager
     
    private void updateList(String name, ArrayList<String> list)
    {
-      listContexts_.get(name).setList(list);
+      ListContext context = listContexts_.get(name);
+      if (context == null)
+      {
+         Debug.logWarning("Unknown workbench list: " + name);
+         return;
+      }
+      context.setList(list);
    }
     
    private class ListContext implements WorkbenchList

